Signal/src/ViewControllers/PhotoLibrary/ImagePickerController.swift

```diff
@@ -52,16 +57,26 @@ class ImagePickerGridController: UICollectionViewController, PhotoLibraryDelegat
 
         collectionView.register(PhotoGridViewCell.self, forCellWithReuseIdentifier: PhotoGridViewCell.reuseIdentifier)
 
-        navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .cancel,
-                                                           target: self,
-                                                           action: #selector(didPressCancel))
+        view.backgroundColor = .ows_gray95
+
+        if let navBar = self.navigationController?.navigationBar as? OWSNavigationBar {
+            navBar.makeClear()
+        } else {
+            owsFailDebug("Invalid nav bar.")
+        }
+
+        let cancelButton = UIBarButtonItem(barButtonSystemItem: .stop,
+                                           target: self,
+                                           action: #selector(didPressCancel))
+        cancelButton.tintColor = .ows_gray05
```

Comment:
In hindsight, I wish I had added "dark..." methods to `Theme`, sort of like you did.